packages/@icon-magic/distribute/test/fixtures/out/entity-backgrounds/entity-backgrounds_company_default_2048x512.imageset/Contents.json

```diff
@@ -0,0 +1,34 @@
+{
+  "images": [
+    {
+      "idiom": "universal",
+      "scale": "1x",
+      "filename": "entity-backgrounds_company_default_2048x512@1.png"
+    },
+    {
+      "idiom": "universal",
+      "scale": "2x",
+      "filename": "entity-backgrounds_company_default_2048x512@2.png"
+    },
+    {
+      "idiom": "universal",
+      "scale": "3x",
+      "filename": "entity-backgrounds_company_default_2048x512@3.png"
+    },
+    {
+      "idiom": "universal",
+      "scale": "1x",
+      "filename": "entity-backgrounds_company_default_on_dark_2048x512@1.png"
+    },
+    {
+      "idiom": "universal",
```

Comment:
Are you going to add `appearances` to the output as part of a different PR?
```
   {
      "appearances" : [
        {
          "appearance" : "luminosity",
          "value" : "dark"
        }
      ],
      "idiom": "universal",
      "scale": "1x",
      "filename": "entity-backgrounds_company_default_on_dark_2048x512@1.png"
    },
```

Reply:
ah forgot about that! will do